signer/src/lib.rs

```diff
@@ -116,7 +122,7 @@ pub const WITHDRAWAL_DUST_LIMIT: u64 = 546;
 /// The value here was taken from the last paragraph of the opening comment
 /// of https://github.com/stacks-network/sbtc/discussions/12 and in the
 /// comments of https://github.com/stacks-network/sbtc/issues/16.
-pub const WITHDRAWAL_MIN_CONFIRMATIONS: u64 = 6;
+pub const WITHDRAWAL_MIN_CONFIRMATIONS: u64 = BITCOIN_CONFIRMATION_DELAY;
```

Comment:
This change and one below are out of scope of this PR, but I think they are pretty small to include them here.